src/lib/Components/Bidding/Elements/Typography.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import styled from "styled-components/native"
+import { color, fontSize, lineHeight, space, textAlign } from "styled-system"
+
+import { Fonts } from "../../../data/fonts"
+
+const Serif = styled.Text`
+  font-family: "${Fonts.GaramondRegular}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+const SerifItalic = styled.Text`
+  font-family: "${Fonts.GaramondItalic}";
+  ${color}
+  ${fontSize}
+  ${lineHeight}
+  ${space}
+  ${textAlign}
+`
+
+export const Serif12 = props => <Serif fontSize={1} {...props} />
```

Comment:
Not sure if this is a "real" concern, but it seems like there are going to be a lot of components that serve only as a translation layer between our design nomenclature and our styled-system primitives. One (really rough) idea that occurred to me is, what if we encoded our font sizes as literal types:

```
type ArtsyFontSize = "1" | "2 | "3" | "3t" | // ...
```

And then, taking advantage of the fact that `fontSize` is effectively a function from props to a chunk of CSS, we could make our own function that wraps that `fontSize`:

```
function artsyFontSize(props: { fontSize: ArtsyFontSize }): { fontSize: number } => string {
  const numericFontSize = // insert conversion logic
  return props => {
    fontSize(numericFontSize)
  }
}
```

This might also help with being able to use more semantically meaningful selectors of our style constants than array indexes.